utils/data.ts

```diff
@@ -97,7 +97,7 @@ export const transactionTabs: TransactionData = {
       lazlo: 'pending',
     },
     input: {
-      sats: '101,000',
+      sats: '100,000',
     },
     output_0: {
       sats: '100000',
```

Comment:
Same as the two comments above since these are all different versions of the same refund transaction :) Should be 99,000 sats